### PR TITLE
Seb test

### PR DESCRIPTION
**Pull request by sebboyer**
Here the three files containing the predicate-like questions. There are in data/
- valid_questions_actions.dictionary  contains rows like 393|0|question 0 craft-fence
- valid_questions_objects.dictionary contains  rows like 336|0|question 1 shears
- valid_questions_predicates.dictionary contains rows like 462|0|question 2 crafting

### CODE REVIEW

**Comment by sebboyer** [comment]
Editing the test config files so that they contain the new questions files.
Starting to add the merging ability of different question files in the LoadPredDictFile function (commented out for now). 
@yala we need to discuss on what criteria add new questions type , and we need to merge the files depending on this criteria in LoadPredDictFile.

**Comment by yala** [comment]
@sebboyer Yeah, it'd be great to chat about this. Call me on Speak whenever

File: data/.ipynb_checkpoints/choosing_predicates-checkpoint.ipynb [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "############ Extract list of objects and actions and predicates\n",
```
Comment (yala) [inline data/.ipynb_checkpoints/choosing_predicates-checkpoint.ipynb:22]: What is this file for?

File: data/choosing_predicates.ipynb [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    "actions=A\n",
+    "objects=O\n",
+    "predicates=P"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {
+    "collapsed": false
```
Comment (yala) [inline data/choosing_predicates.ipynb:36]: What's this file for?

Do we need it in the repo?

File: data/valid_predicates.dictionary [hunk 3]
```diff
@@ -334,3 +334,4 @@
 333|4|thing-available clayblock
 334|5|thing-available clayblock
 335|6|thing-available clayblock
+336|1|question-object clayblock
```
Comment (yala) [inline data/valid_predicates.dictionary:337]: This line shouldn't be here

File: code/model/SubgoalPolicy.cpp [hunk 4]
```diff
@@ -849,6 +849,18 @@ bool SubgoalPolicy::LoadPredDictFile (void)
 {
 	String sPddlDictFile = (config)"pddl_dict_file";
 	cout << "   loading dict: " << sPddlDictFile << endl;
+
+	// String sPddlDictFile_init = (config)"pddl_dict_file";
+	// cout << "   loading dict: " << sPddlDictFile_init << endl;
+	// String sPddlDictFile_actions = (config)"pddl_dict_question_actions_file";
+	// cout << "   loading dict: " << sPddlDictFile_actions << endl;
+	// String sPddlDictFile_objects = (config)"pddl_dict_question_objects_file";
+	// cout << "   loading dict: " << sPddlDictFile_objects << endl;
```
Comment (yala) [inline code/model/SubgoalPolicy.cpp:858]: Let's meet and chat about this, i think this is the right direction

**Pull request merged**